code/modules/clothing/clothing.dm

```diff
@@ -46,6 +46,8 @@
 	var/limb_integrity = 0
 	/// How many zones (body parts, not precise) we have disabled so far, for naming purposes
 	var/zones_disabled
+	/// If this can be eaten by a moth
+	var/moth_edible = TRUE
```

Comment:
Make this an element or something, snowflake bad

Reply:
Could you explain what would be best here? I've worked on components before but not elements, so I'm not sure how to convert this without just making it a more complicated variable.

Reply:
Elements are basically singleton components

Reply:
I've been looking into this for the last few days and I really can't see a way to make it work as an element. Every approach I've tried seemed to have some sort of inherent flaw.

The best I've managed was fully converting moth clothes eating into a 'species-specific item eating' element, but that ended up being almost twice the size that this PR is currently, required edits to SSdcs (would need a lot of testing), and really was just out of the scope of this PR.

I definitely agree that `moth_edible` is a bit snowflakey though, so I'll try to find another way to do it.